.github/workflows/release-please.yml

```diff
@@ -26,3 +29,100 @@ jobs:
             VERSION
             barretenberg.nix
             barretenberg-wasm.nix
+
+  build-linux:
+    name: Build on Ubuntu
+    needs: [release-please]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install Bleeding edge CMake
+        run: |
+              sudo apt -y remove --purge cmake
+              sudo snap install cmake --classic
+      - name: Install Yarn
+        run: |
+          curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+          echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+          sudo apt -y update && sudo apt -y install yarn
+      - name: Install Clang/Ninja/Binaryen
```

Comment:
We don't need binaryen anymore.
Was this copied from previous nix stuff as i see nix related files above as well?